### DISCUSSION TITLE
Advanced Product Fields - App or Plugin?

### DISCUSSION THREAD

**Discussion by guyathomas** [saleor/saleor#8645] [Ideas]
I'm considering contributing to saleor to support a use case similar to this https://www.studiowombat.com/plugin/advanced-product-fields-for-woocommerce/

Where
* A product can have additional extra's that will determine the price
* A product's price will be based on the width and length of the user input (either a linear equation or a CSV lookup table)

Would you see this is more appropriate for an app or a plugin - or maybe even best sits in the core code? This distinction on the tradeoffs / use cases for both is not super clear to me.

**Comment by zakton5** [comment]
This is something I need as well. Saleor looks like an incredible platform but without this functionality it will not work for us.
